views/default/js/elgg/admin.js

```diff
@@ -181,6 +190,51 @@ define(function(require) {
 			}
 		}
 	};
+	
+	/**
+	 * Filters the plugin list based on a selected category
+	 *
+	 * @return void
+	 */
+	function filterPluginCategory (e) {
+		e.preventDefault();
+		
+		// remove selected state from all buttons
+		$(".elgg-admin-plugins-categories li").removeClass("elgg-state-selected");
+	
+		// show plugins with the selected category
+		$(".elgg-plugin").hide();
+		$(".elgg-plugin-category-" + $(this).attr("rel")).show();
+		$(this).parent().addClass("elgg-state-selected");
+	};
+	
+	/**
+	 * Shows the details of the selected plugin
+	 *
+	 * @return void
+	 */
+	function showPluginDetails () {
+		// remove selected state from all buttons
+		$(".elgg-plugin-details-container > ul > li").removeClass("elgg-state-selected");
+		
+		$(".elgg-plugin-details-container > div > div").hide();
```

Comment:
Can't use `hidden` property (and probably not "hidden" class) because this relies on elgg-tabs JS functionality.